env.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
This script is pretty clever, but it seems a little complex for setting 3 values in a config file.  Could we not just have a template `env.json` with placeholders, and do a `sed` find/replace like we do in [`keycloak/docker-entrypoint.sh`](https://github.com/cfpb/hmda-platform-auth/blob/master/keycloak/docker-entrypoint.sh).

If you do want to keep it this way, I think it at least deserves some comments describing the intent of the script.